### PR TITLE
Enable Prow for Auth

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

**Comment by sebastienvas** [comment]
Prow presubmit available at https://k8s-gubernator.appspot.com/build/istio-prow/pull/istio_auth/190/auth-presubmit/9/

File: prow/auth-presubmit.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+if [ "${CI:-}" == 'bootstrap' ]; then
+    # Test harness will checkout code to directory $GOPATH/src/github.com/istio/istio
```
Comment (nlandolfi) [inline prow/auth-presubmit.sh]: istio/auth

File: prow/auth-presubmit.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    # but we depend on being at path $GOPATH/src/istio.io/istio for imports.
```
Comment (nlandolfi) [inline prow/auth-presubmit.sh]: istio/auth

**Comment by lookuptable** [comment]
I don't see `prow/auth-presubmit.sh` shows in the list of checks. Is it expected or we need to enable it somewhere?

screen shot 2017-07-26 at 12 40 00 pm

**Comment by nlandolfi** [comment]
@lookuptable we will enable it by removing [this line](https://github.com/istio/test-infra/blob/master/prow/config.yaml#L34) after this PR is merged and the prow/auth-presubmit file is in the master branch (otherwise we would have failing presubmits on all other PRs for this repo during the course of introducing this job).

**Pull request merged**